cmake/configure/configure_VTK.cmake

```diff
@@ -0,0 +1,27 @@
+# This file is part of 4C multiphysics licensed under the
+# GNU Lesser General Public License v3.0 or later.
+#
+# See the LICENSE.md file in the top-level for license information.
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+find_package(VTK REQUIRED COMPONENTS CommonCore CommonDataModel IOXML)
+
+if(VTK_FOUND)
+  message(STATUS "VTK component libraries: ${VTK_LIBRARIES}")
+
+  target_compile_definitions(
+    VTK::CommonCore
+    INTERFACE "-DVTK_MAJOR_VERSION=${VTK_MAJOR_VERSION}"
+              "-DVTK_MINOR_VERSION=${VTK_MINOR_VERSION}"
+              "-DVTK_BUILD_VERSION=${VTK_BUILD_VERSION}"
+    )
```

Comment:
I think we also have this hack in other places but now I am wondering if this can ever work when using an installed version 🤔 It seems we are not adding these definitions in this case.